### PR TITLE
Enabling KaTex Instructions

### PR DESCRIPTION
**Pull request by justingwl**
Hi - This would have been really helpful if that was documented, many users aren't able to render the KaTex and aren't sure why/where to look
Also,  fixed spelling some spelling.
Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,17 @@ disableKinds = ['taxonomy', 'taxonomyTerm']
   BookServiceWorker = true
 ```
 
+
+### KaTex
+By default, Goldmark doees not render KaTex. To enable it, append the following to your ```config.toml``` file.
```
Comment (alex-shpak) [inline README.md:197]:
```suggestion
By default, Goldmark does not render KaTex. To enable it, append the following to your ```config.toml``` file. 
```

**Comment by alex-shpak** [comment]
Cool 👍 
I think it can be extended not to just `KaTeX`, in general Goldmark will strip output that doesn't look safe. I would suggest to move that text under `Shortcodes` section.

**Comment by justingwl** [comment]
KaTex seems to be the only element that would not render for me, what else is Goldmark not rendering in the project?

**Comment by alex-shpak** [comment]
Yeah, right now KaTeX is probably only one suffering from trimming unsafe output. But issue is rather more generic and affects any suspicious output 🤔 I think mermaid might be also affected.

**Pull request merged**